src/angularfire2.ts

```diff
@@ -2,8 +2,9 @@ import * as firebase from 'firebase/app';
 import * as utils from './utils';
 import { FirebaseAppConfigToken, FirebaseApp, _firebaseAppFactory } from './app/index';
 import { FirebaseListFactoryOpts, FirebaseObjectFactoryOpts, FirebaseAppConfig } from './interfaces';
-import { FirebaseAppName } from './tokens';
-import { Injectable, OpaqueToken, NgModule } from '@angular/core';
+import { Injectable, InjectionToken, OpaqueToken, NgModule } from '@angular/core';
```

Comment:
Hi @davideast I think that's quite weird.
TypeScript should not require you to import this just because `OpaqueToken` is used somewhere else.
If you look into the generated code in the `next` package on npm, `OpaqueToken` is not in the import list, as it has been tree-shaken I assume, a lot of other imports are removed as well
```
import { FirebaseAppConfigToken, FirebaseApp, _firebaseAppFactory } from './app/index';
import { InjectionToken, NgModule } from '@angular/core';
var FirebaseAppName = new InjectionToken('FirebaseAppName');
```

I forked master and removed some imports, tests complete successfully and the build is also successful (producing the same )
Should I open an issue/PR for this? Or am I overlooking something?